src/components/__tests__/BottomNavigation.test.js

```diff
@@ -334,8 +334,6 @@ it('renders custom background color passed to barStyle property', () => {
 
   const wrapper = getByTestId('bottom-navigation-bar-content');
   expect(wrapper.props.style).toEqual(
```

Comment:
Custom matcher from `@testing-library/jest-native`:
```suggestion
  expect(wrapper).toHaveStyle({ backgroundColor: red300 });
```